infra/concourse/pipelines/terraform-google-jenkins.yml

```
@@ -0,0 +1,130 @@
+resource_types:
+
+  - name: pull-request
+    type: docker-image
+    source:
+      repository: teliaoss/github-pr-resource
+
+resources:
+  - name: pull-request
+    type: pull-request
+    webhook_token: ((jenkins.github_webhook_token))
+    source:
+      repository: terraform-google-modules/terraform-google-jenkins
+      access_token: ((github.pr-access-token))
+
+  - name: lint-test-image
+    type: docker-image
+    source:
+      repository: gcr.io/cloud-foundation-cicd/cft/lint
+      username: _json_key
+      password: ((sa.google))
+
+  - name: integration-test-image
+    type: docker-image
+    source:
+      repository: gcr.io/cloud-foundation-cicd/cft/kitchen-terraform
+      tag: 0.11.10_216.0.0_1.19.1_0.1.10
```

This should be updated to the latest image.

```suggestion
      tag: 1.2.0
```